extensions-core/datasketches/src/main/java/io/druid/query/aggregation/datasketches/tuple/ArrayOfDoublesSketchToEstimateAndBoundsPostAggregator.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query.aggregation.datasketches.tuple;
+
+import java.util.Comparator;
+import java.util.Map;
+import java.util.Objects;
+
+import javax.annotation.Nullable;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.yahoo.sketches.tuple.ArrayOfDoublesSketch;
+
+import io.druid.java.util.common.IAE;
+import io.druid.query.aggregation.AggregatorUtil;
+import io.druid.query.aggregation.PostAggregator;
+
+/**
+ * Returns a distinct count estimate and error bounds from a given {@link ArrayOfDoublesSketch}."
+ * The result will be three double values: estimate of the number of distinct keys, lower bound and upper bound.
+ * The bounds are provided at the given number of standard deviations (optional, defaults to 1).
+ * This must be an integer value of 1, 2 or 3 corresponding to approximately 68.3%, 95.4% and 99.7%
+ * confidence intervals.
+ */
+public class ArrayOfDoublesSketchToEstimateAndBoundsPostAggregator extends ArrayOfDoublesSketchUnaryPostAggregator
+{
+
+  private final int numStdDevs;
+
+  @JsonCreator
+  public ArrayOfDoublesSketchToEstimateAndBoundsPostAggregator(
+      @JsonProperty("name") final String name,
+      @JsonProperty("field") final PostAggregator field,
+      @JsonProperty("numStdDevs") @Nullable final Integer numStdDevs
+  )
+  {
+    super(name, field);
+    this.numStdDevs = numStdDevs == null ? 1 : numStdDevs;
+  }
+
+  @JsonProperty
+  public int getNumStdDevs()
+  {
+    return numStdDevs;
+  }
+
+  @Override
+  public Object compute(final Map<String, Object> combinedAggregators)
```

Comment:
Please change the return type to `double[]` for type safety.

Reply:
ok